### DISCUSSION TITLE
TV Shows and Animes doesn't appear as available even if they are

### DISCUSSION THREAD

**Discussion by Nehrz** [seerr-team/seerr#682] [Q&A] [answered]
Hello,

Concerning movies, everything works as intended : request > movie added > movie appears as available and a link to watch on jellyfin.

Now for tv shows and animes : request > show/anime added > show/anime still appears as requested even though it is available on jellyfin.

Is this a naming problem ?
I have TV Shows/My show name/My show name (year) - S01E01 - episode name [video quality][audio codec][video codec].mkv

**Comment by fallenbagel** [comment, marked as answer]
Thats a configuration issue. Flat season structure is not supported in 1.7.0. That support will be on 1.8.0 (its currently live on develop). For 1.7.0 you need them to be in season folders (for example, Season 01/Season 1), and for new requests make sure services > sonarr > season folders is ticked.

**Reply by Nehrz** [reply]
It works. Thank you very much !
